mmdet/apis/train.py

```diff
@@ -107,6 +108,7 @@ def train_detector(model,
     runner.register_training_hooks(cfg.lr_config, optimizer_config,
                                    cfg.checkpoint_config, cfg.log_config,
                                    cfg.get('momentum_config', None))
+    runner.register_hook(CompatibleCheckHook())
```

Comment:
The function in train.py will be unified in the future. May use customized hook and add it in the default_runtime.py